package.json

```diff
@@ -140,6 +141,7 @@
     "lage": "0.27.0",
     "lerna": "^3.21.0",
     "lint-staged": "^10.2.9",
+    "node-fetch": "2.6.1",
```

Comment:
Was moved to root from `scripts/` and `fluentui/perf-test`, also removed carets in versions